lib/geordi/gitpt.rb

```diff
@@ -96,4 +119,39 @@ def bold(string)
     HighLine::BOLD + string + HighLine::RESET
   end
 
+  def checkout_branch(name, new_branch: false)
+    if new_branch
+      Geordi::Util.system! 'git', 'checkout', 'master'
+      Geordi::Util.system! 'git', 'checkout', '-b', name
+    else
+      Geordi::Util.system! 'git', 'checkout', name
+    end
+  end
+
+  def normalize_branch_name(name)
+    name.gsub!('ä', 'ae')
+    name.gsub!('ö', 'oe')
+    name.gsub!('ü', 'ue')
+    name.gsub!('ß', 'ss')
+    name.tr!('^A-Za-z0-9_ ', '')
+    name.squeeze! ' '
+    name.gsub!('-', '_')
+    name.gsub!(' ', '_')
```

Comment:
Our convention is to use dashes in branch names, so please change the code accordingly.
L138 is not needed as dashes are stripped in L136.